ubiquibot-config-default.json

```diff
@@ -87,6 +87,7 @@
     }
   },
   "enable-access-control": {
-    "label": false
+    "label": false,
+    "organization": true
```

Comment:
All of the bot's features should be disabled by default, it was discussed today somewhere in telegram :)

Reply:
I wish that the property name was more clear. When I saw this in `ubiquity/ubiquibot-config/.github/ubiquibot-config.yml` I had to trace this down to this pull request to understand what it does. 

Perhaps we can rename to: 

```
  "public-access-control": {
     "set-label": false,
     "fund-external-closed-issue": false
```

What do you guys think?

Reply:
Those new params were introduced solely for the demo purposes for the `Ethcon Korea 2023` conference

I would simply delete those new params because:
1. I can't image a case when a partner wants to tweak one of those params
2. If, by chance, partner forgets to set those new params to `true` then it poses a significant security issue

@0xcodercrane What do you think?

Reply:
I like the idea of being proactive about the security for our users, but I feel like they really have to go out of their way to dig up this config property. In our documentation we could just make it super clear that it is a significant security risk to enable them, but, there might be reasons why other organizations want these enabled?

Reply:
> In our documentation we could just make it super clear that it is a significant security risk to enable them

On the contrary, it is a significant security risk to disable [label](https://github.com/ubiquity/ubiquibot-config/blob/694633a8e6f2d5ae7ffe23f97f625c354737dca2/.github/ubiquibot-config.yml#L81) and [organization](https://github.com/ubiquity/ubiquibot-config/blob/694633a8e6f2d5ae7ffe23f97f625c354737dca2/.github/ubiquibot-config.yml#L82) access controls

Anyway, I feel like that in the future (with 0 unit testing) we will get a security issue related to one of those params

Reply:
Agree to rename the property names: I had the same feeling to guess what the property does. took a little bit

Reply:
> Anyway, I feel like that in the future (with 0 unit testing) we will get a security issue related to one of those params

Then I guess lets label the commit, remove it from the codebase, and restore the capability based on the commit label in the future if we need it again. 

Alternatively we could have a "demo" instance of the bot because I plan to be presenting it a lot more later this year and using the same demo script (order a bag or shirt) would be fantastic

Reply:
>  I plan to be presenting it a lot more later this year

And we're getting the case for those access control params. Ok then, let's simply [rename](https://github.com/ubiquity/ubiquibot/issues/742) them for now.